src/xbps/packages/downgrading.md

```diff
@@ -0,0 +1,20 @@
+# Downgrading to a specific package version
+
+The easiest way to downgrade is by using `xdowngrade` from the `xtools` package.
+
```

Comment:
Please make a clear separation, making it clear this is the end of this subsection about `xdowngrade`. Maybe there is a bit of usage you can duplicate into here too?

Reply:
Thanks for the feedback - PR updated.